ethcore/src/verification/verification.rs

```diff
@@ -281,6 +281,11 @@ pub fn verify_header_params(header: &Header, engine: &EthEngine, is_full: bool,
 	if header.gas_limit() < &min_gas_limit {
 		return Err(From::from(BlockError::InvalidGasLimit(OutOfBounds { min: Some(min_gas_limit), max: None, found: header.gas_limit().clone() })));
 	}
+	if let Some(limit) = engine.maximum_gas_limit() {
+		if header.gas_limit() > &limit {
+			return Err(From::from(::error::BlockError::InvalidGasLimit(OutOfBounds { min: None, max: Some(limit), found: header.gas_limit().clone() })));
```

Comment:
`clone()` on copy-type 
```suggestion
			return Err(From::from(::error::BlockError::InvalidGasLimit(OutOfBounds { min: None, max: Some(limit), found: *header.gas_limit() })));
```